### DISCUSSION TITLE
Kernel versions

### DISCUSSION THREAD

**Discussion by ajgillette** [sustainable-computing-io/kepler#745] [Q&A] [answered]
Hey Team,
I am building Kepler from a checkout and deploying on a cluster where the nodes are running a 6.3.5-1.el7.elrepo.x86_64 kernel.
When I exec into the Kepler-exporter pod, it is has the 5.14.0-284.11.1.el9_2.x86_64 headers and modules.  

When I look at the logs from the kepler-exporter, there is an error saying that the container can not find the headers for 5.14.0-284.11.1.el9_2.x86_64. 

That kernel is not running anywhere in the cluster or on the node where I built the source.

How to I fix this? Is there a config file in the source tree?

**Comment by rootfs** [comment, marked as answer]
@ajgillette can you paste the kepler logs? `kubectl logs -n kepler daemonset/kepler-exporter` will do the trick

**Reply by ajgillette** [reply]
No need this has been addressed :)
